Week3/homework/step2-3.js

```diff
@@ -6,9 +6,13 @@ function repeatStringNumTimesWithFor(str, num) {
   let result = '';
 
   // Replace this comment and the next line with your code
-  console.log(str, num, result);
-
-  return result;
+  if (num > 0) {
+    for (let i = 0; i < num; i++) {
+      result += str;
+    }
+    return result;
+  }
+  return '';
 }
```

Comment:
👍 but you can also simply return `result` at the end of the function and need not return anything then inside `if` block. It's just one less line of code 😄 

```
  let result = '';
  if (num > 0) {
    for (let i = 0; i < num; i++) {
      result += str;
    }
  }
  return result;
```